templates/spin_sidebar.hbs

```diff
@@ -64,6 +64,7 @@
             Tools
         </p>
         <ul class="menu-list">
+            <li><a {{#if (active_content request.spin-path-info "/spin/managing-cache" )}} class="active" {{/if}} href="{{site.info.base_url}}/spin/managing-cache">Managing Cache</a></li>
```

Comment:
Suggest in the Advanced section rather than Tools (if we do keep a separate page rather than merge onto the registry page).

Reply:
The advanced section can work, yes!

By merging into the registry page, are we talking about tapping into a section like [this current documentation](https://developer.fermyon.com/spin/distributing-apps.md#running-published-applications-by-digest) so that we can mention the benefits of having a registry cache for the fetched resources? If so, that is a great idea (perhaps a brief note that mentions/links the advanced section as having more information on managing the registry cache).

Reply:
Please check the latest commit for updates.